src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -613,23 +613,23 @@ mlir::Type ONNXOpsDialect::parseType(mlir::DialectAsmParser &parser) const {
 
     if (parser.parseGreater())
       return Type();
-    return SeqType::get(elementTypes);
-  } else {
-    llvm_unreachable("Unexpected onnxmlir keyword");
```

Comment:
else with llvm_unreachable has been removed in favour of emitting a more informative parse error via `parser.emitError`.